lib/providers/models/explore_card_filters_model.dart

```diff
@@ -2,12 +2,25 @@ import 'package:flutter/material.dart';
 import 'package:mm_flutter_app/__generated/schema/schema.graphql.dart';
 
 class ExploreCardFiltersModel extends ChangeNotifier {
-  static const List<String> countries = ['USA'];
-  static const List<String> languages = ['English', 'Urdu', 'Hindi'];
-  static const List<String> skills = ['Marketing', 'Operations', 'StartingUp'];
+  final List<String> countries;
+  final List<String> languages;
+  final List<String> skills;
```

Comment:
This should probably be renamed to `expertises` throughout the app, for consistency. Not sure where else `skills` is used or whether that's in scope for this PR.

Reply:
Done. I've replaced it in some other places throughout this PR.